var/spack/repos/builtin/packages/hbase/package.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2013-2019 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hbase(Package):
+    """
+    Apache HBase is an open-source, distributed, versioned, column-oriented
+    store modeled after Google' Bigtable: A Distributed Storage System for
+    Structured Data by Chang et al. Just as Bigtable leverages the distributed
+    data storage provided by the Google File System, HBase provides
+    Bigtable-like capabilities on top of Apache Hadoop.
+    """
+
+    homepage = "https://github.com/apache/hbase"
+    url      = "https://github.com/apache/hbase/archive/rel/1.4.12.tar.gz"
```

Comment:
```suggestion
    url      = "https://github.com/apache/hbase/archive/rel/2.2.2.tar.gz"
```

Reply:
fix this in 
https://github.com/spack/spack/pull/14161/commits/ba9de2ffd5f9afe1307b3e3be23414efe779b6d9